src/test/java/racing/RacingCarTest.java

```diff
@@ -0,0 +1,14 @@
+package racing;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class RacingCarTest {
+    @Test
```

Comment:
```suggestion
    @Test
    @DisplayName("...")
```

1~2단계에서 학습했던 displayName을 사용해보는건 어떨까요?